### ISSUE TITLE
Tutorial - Stuck on Mammoths obcjetive: Giving something back to the world

### ISSUE THREAD

**Issue by Yuuhi84** [PrehistoricKingdom/feedback#594]
Hi,
During the Breed Mammoths in the incubator, the tutorial can be bugged by prematurely raising let's say, two mammoths and releasing them into the enclosure before the tutorial directs you later, after raising the rest of the mammoths and releasing them into the enclosure, only the mammoths that are released at this stage are counted in the quest, and the previously released ones are not counted.
As a result, the whole tutorial gets bugged and you can't do anything except start again or give up completely.

Best
Yuuhi
 :
![20220428183949_1](https://user-images.githubusercontent.com/104585776/165802410-afb9885c-9541-404d-9623-ddcc316744d0.jpg)
)

**Comment by IIAnnaBananaII** [comment]
Same problem my end :)

**Comment by JeremyCuddles** [comment]
Yeah, I incubated one mammoth, set it in the enclosure. Wondered why it wasn't progressing. Made the other 4 mammoths. Now I am stuck at 4/5. Lol. Oh well.

**Comment by AlienDilo** [comment]
Happened to me aswell, I had bred two before I realized I had to breed five. It still lets you release them which is weird.

**Comment by HatTheBat** [comment]
Same here, can't breed more or select anything else to fix it.
